tests/next_tests/unit_tests/program_processor_tests/codegens_tests/gtfn_tests/test_gtfn_module.py

```diff
@@ -71,3 +88,103 @@ def test_codegen(fencil_example):
     assert module.entry_point.name == fencil.id
     assert any(d.name == "gridtools_cpu" for d in module.library_deps)
     assert module.language is languages.CPP
+
+
+def test_hash_and_diskcache(fencil_example):
+    fencil, parameters = fencil_example
+    compilable_program = stages.CompilableProgram(
+        data=fencil,
+        args=arguments.CompileTimeArgs.from_concrete_no_size(
+            *parameters, **{"offset_provider": {}}
+        ),
+    )
+    hash = gtfn.fingerprint_compilable_program(compilable_program)
+    path = tempfile.gettempdir()
```

Comment:
FYI: Were you aware of the [tmp_path](https://docs.pytest.org/en/6.2.x/tmpdir.html) fixture?

Reply:
No, I wasn't. Thanks for pointing it out, now I am using it.